ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldLRspec-CUI-nonprod.json

```diff
@@ -233,3 +233,46 @@
     ]
   }
 ]
+
+[
```

Comment:
i think if you are adding authorisation you need to put it in the same array as the other fields for authorisation